likecoin/admin/metabox.php

```diff
@@ -189,12 +205,15 @@ function likecoin_get_meta_box_publish_params( $post, $force = false ) {
 /**
  * Add the publish session of likecoin widget metabox
  *
- * @param object| $publish_params Params for displaying publish related settings.
+ * @param object|  $publish_params Params for displaying publish related settings.
+ * @param WP_Post| $post Post object.
  */
-function likecoin_add_publish_meta_box( $publish_params ) {
-	$iscn_hash   = $publish_params['iscn_hash'];
-	$status      = likecoin_parse_publish_status( $publish_params );
-	$iscn_status = likecoin_parse_iscn_status( $publish_params );
+function likecoin_add_publish_meta_box( $publish_params, $post ) {
+	$iscn_hash      = $publish_params['iscn_hash'];
+	$status         = likecoin_parse_publish_status( $publish_params );
+	$arweave_status = likecoin_get_post_arweave_status( $post );
+	$iscn_status    = likecoin_parse_iscn_status( $publish_params );
+	$content        = likecoin_filter_matters_post_content( $post );
```

Comment:
Where is `$content` used in this function?

Reply:
True, it's not used. Will delete it.